tsconfig.json

```diff
@@ -1,28 +1,23 @@
 {
+  "extends": "@tsconfig/react-native/tsconfig.json",
```

Comment:
faced a warning `File '@tsconfig/react-native/tsconfig.json' not found.ts`

Reply:
there might be a ts/node_modules cache, if you remove these and re-install and re-invalidate the cache. I think this should be solved automatically.